Documentation/rust/coding-guidelines.rst

```diff
@@ -39,6 +39,69 @@ individual files, and does not require a kernel configuration. Sometimes it may
 even work with broken code.
 
 
+Comments
+--------
+
+"Normal" comments (i.e. ``//``, rather than code documentation which starts
+with ``///`` or ``//!``) are written in Markdown the same way as documentation
+comments are, even though they will not be rendered. This improves consistency,
+simplifies the rules and allows to move content between the two kinds of
+comments more easily. For instance::
+
+.. code-block:: rust
+
+	// Skip one since `v` is not cleared.
```

Comment:
It's better to put code that actually uses `v` in the example, thoughts?

Reply:
I could put a different example so that we stay within 2 lines.

Reply:
Done.